WeatherMan.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import sys
+
+from DataReader import data_reader
+from View import View
+
+weather_report = {}
+
+data_reader = data_reader()
+view = View()
+
+arg_parser = argparse.ArgumentParser()
+arg_parser.add_argument("[report#]", type = int, default = 0, help = 'report number')
+arg_parser.add_argument("[data_dir]", type = str, default = "none", help = 'data directory')
```

Comment:
You can provide a callback to `type` param that should check whether it is a valid directory.